examples/using-mobx/readme.md

```diff
@@ -9,10 +9,13 @@ Gatsby example site that shows use of mobx.
 To use mobx in a Gatsby site you'll need to hook in to two of Gatsby's extension points.
 
 Once in wrapRootElement which runs during Gatsby's server rendering process, and once in wrapRootElement which is part of Gatsby's browser APIs.
-Check out [./gatsby-ssr.js](./gatsby-ssr.js) and [./gatsby-browser.js](./gatsby-browser.js) to see how this is implemented in this example
 
-## More Information on MobX
+Check out [./gatsby-ssr.js](./gatsby-ssr.js) and [./gatsby-browser.js](./gatsby-browser.js) to see how this is implemented in this example.
 
-[Docs](https://mobx.js.org/)
+## More information
 
-[Examples](https://github.com/mobxjs/mobx-examples)
+[mobx-react-devtools](https://github.com/mobxjs/mobx-react-devtools) package is now deprecated and was removed. As it could generate [this error](https://github.com/mobxjs/mobx-react-devtools/issues/117).
```

Comment:
```suggestion
[mobx-react-devtools](https://github.com/mobxjs/mobx-react-devtools) package is now deprecated and was removed. As it could generate the error [Cannot read property 'on' of undefined](https://github.com/mobxjs/mobx-react-devtools/issues/117).
```

Reply:
Thanks for the revision. I agree with the error part, but why the removal of the examples?

Reply:
I am confused. I just suggested to change the link title to have "talkig links".

Reply:
No you're right i read it wrong, the examples part was my bad, when updating the readme i accidentally deleted the examples part. I'm currently changing the other part accordingly and going to push it in a bit